sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/HIFIOWithEmbeddedCassandraTest.java

```diff
@@ -38,12 +38,14 @@
 import org.apache.hadoop.mapreduce.InputFormat;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 /** Tests to validate HadoopInputFormatIO for embedded Cassandra instance. */
+@Ignore("Ignored because HadoopInputFormatIO is deprecated, tested by HadoopFormatIOCassandraTest")
```

Comment:
If it is already tested by the other test, why not remove it entirely?

Reply:
Well, actually the whole module `hadoop-input-format` should be removed. I left it "just in case". Anyway, it won't be triggered to run.